Makefile

```diff
@@ -4,15 +4,19 @@ GIT_VERSION ?= $(shell git describe --tags --always --dirty)
 
 LDFLAGS=-buildid= -X github.com/gittuf/gittuf/internal/version.gitVersion=$(GIT_VERSION)
 
-.PHONY : build test install
+.PHONY : build test install fmt
 
 default : install
 
 build : test
-	CGO_ENABLED=0 go build -trimpath -ldflags "$(LDFLAGS)" .
+	mkdir -p ./dist
+	CGO_ENABLED=0 go build -trimpath -ldflags "$(LDFLAGS)"  -o dist/gittuf .
 
-install : test
+install : test 
```

Comment:
nit: I think the only diff on this line now is the trailing space, please remove that.

Reply:
Fixed